phpcs.xml

```diff
@@ -17,5 +17,8 @@
     <file>benchmark</file>
 
     <!-- Include all rules from the Laminas Coding Standard -->
-    <rule ref="LaminasCodingStandard"/>
+    <rule ref="LaminasCodingStandard">
+        <!-- Exclude declare(strict_types=1) until the next major release. This component heavily depends on type-magic and thus should avoid adding strict_types as of now v3 -->
```

Comment:
If we roll a new major release, it would be fine to remove this exclusion.

As the current patch is presented, a new major version is indeed needed.

Reply:
If we do major release we will need to add it into "doctrine/doctrine-module" it depends on v3 ->   "laminas/laminas-cache": "^3.1.2".

Let me know what is final decision on this and I will update it accordingly.

Reply:
Decided not to roll with a new major: resolving here.